.circleci/config.yml

```diff
@@ -7,23 +7,42 @@ steps: &steps_spec
       command: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle
   - run:
       command: bundle exec rake
-  - run:
-      command: bundle exec rake benchmarks
+  - when:
+      condition: << parameters.benchmark >>
+      steps:
+        - run:
+            command: bundle exec rake benchmarks
 
 jobs:
-  ruby-2.2:
+  ruby-2_2:
+    parameters:
+      benchmark:
+        type: boolean
+        default: false
```

Comment:
passing in benchmark parameters here to false, while the other ruby versions will be true.